src/hooks/useBalanceChartData/useBalanceChartData.ts

```diff
@@ -27,22 +26,21 @@ import {
   PortfolioBalancesById,
 } from 'state/slices/portfolioSlice/portfolioSliceCommon'
 import {
-  selectAccountSpecifiers,
   selectAssets,
+  selectBalanceChartCryptoBalancesByAccountIdAboveThreshold,
   selectCryptoPriceHistoryTimeframe,
   selectFiatPriceHistoriesLoadingByTimeframe,
   selectFiatPriceHistoryTimeframe,
   selectPortfolioAssets,
-  selectPortfolioCryptoBalancesByAccountIdAboveThreshold,
   selectPriceHistoriesLoadingByAssetTimeframe,
-  selectTotalStakingDelegationCryptoByAccountSpecifier,
+  selectRebasesByFilter,
+  selectTxHistoryStatus,
   selectTxsByFilter,
 } from 'state/slices/selectors'
-import { selectRebasesByFilter } from 'state/slices/txHistorySlice/selectors'
 import { Tx } from 'state/slices/txHistorySlice/txHistorySlice'
 import { useAppSelector } from 'state/store'
 
-import { includeStakedBalance, includeTransaction } from './cosmosUtils'
+import { includeTransaction } from './cosmosUtils'
```

Comment:
we still need this utility and can't handle this at the selector layer, as we need to consider the *fees* for these transactions, but not act on adjusting the *balances* due to these transactions.

Reply:
While at it, can we do a bit of renaming here?
e.g `shouldIncludeTransaction` on the consumption.
Not sure about the naming of the util itself, I would probably invert all the logical checks and call it `isCosmosTransaction`

Reply:
Done in 96e442d36c0458cd10e81152e10a5c1a77cf3ae2 and cce5f1df78a75f3b8603404a3f088092ae8b267d